### DISCUSSION TITLE
Book page number on the side

### DISCUSSION THREAD

**Discussion by matejdro** [koreader/koreader#8080] [Q&A] [answered]
I've noticed some epubs display page numbers in the right margin of my koreader:

![image](https://user-images.githubusercontent.com/507922/129474701-3bc26738-96ae-4930-9231-81f9dc9ea263.png)

Interestingly enough, with every subsequent page, number moves a bit lower, until it eventually wraps around back on top.

Is this just how some epubs are formatted, or is there a way to remove this?

**Comment by Frenzie** [comment, marked as answer]
Probably related to https://github.com/koreader/koreader/pull/6847 or something (that is, those would be the physical page numbers).

**Reply by NiLuJe** [reply]
Yep, those are the ePub 3 page maps ;).

EDIT: #6004

**Reply by matejdro** [reply]
Yup, #6004 is the one, disabling it removes the numbers.

Thanks a lot!
